my-app/src/index.css

```
@@ -0,0 +1,142 @@
+html,
+body,
+#root {
+  margin: 0;
+
+  padding: 0;
+
+  height: 100%;
+}
+
+#root {
+  height: 100%;
+  display: flex;
+}
+
+.TypingArea {
```

```suggestion
.typing-area {
```

Please adjust everywhere.